app/presenters/statistics_presenter.rb

```diff
@@ -31,7 +31,7 @@ def version
   end
 
   def open_registrations?
-    AppConfig.settings.enable_registrations
+    AppConfig.settings.enable_registration?
```

Comment:
Ok, are you willin to continue asking me riddles or will you eventually help me ?
[You](https://github.com/diaspora/diaspora/pull/5599#discussion_r23799476) told me tu put a `?` at the end of that line and I did and it doesn't work. So I'm sorry if I'm missing something but right now, I've got better to do than playing puzzles.